README.md

```diff
@@ -250,6 +250,8 @@ Usage: `./duplicate-files <source_path> <n> <destination_path (optional)>`
 
 *You can change the number to reduce or increase the number of files and consequently the resulting size.
 
+*NB: The files in `./test-tree/misc/variety/files.zip` where originally obtained from Permanent aws via `aws s3 cp s3://permanent-repos/test_files/critical-path.zip`. (Access required via aws authentication)*
```

Comment:
"were originally obtained"  (just typo: "where" --> "were")

Reply:
And I would recommend 

* Explicitly mentioning that these are the same files that https://github.com/PermanentOrg/functional-test uses
* Give the date on which the files were accessed, so that it's easier to notice later if there's divergence.  (Yes, I know the version control history in Git shows when the files were *added*, but that's not necessarily the same as the date they were *retrieved* from AWS, and the latter is the important date here).

Reply:
@kfogel grammar fixed (entire text rephrased), and functional test repo referenced however, I am not certain about when the files where access, I am assuming it's probably the same day I added. So, I am rather mentioning that, the reader can check for changes by looking at the original source if they want to stay in sync with the original source.

Also, these files could be any bunch of mixed files that we expect  that users would typically have (images, documents, videos etc) it has no technical link or dependence the functional-test or the aws data (at least not anymore).... That said, I now mentioned that those files might have changed in their original source and the reader can check if they want to.